packages/types/src/block-kit/blocks.ts

```diff
@@ -130,15 +132,11 @@ export interface HeaderBlock extends Block {
  * @description Displays an image. A simple image block, designed to make those cat photos really pop.
  * @see {@link https://api.slack.com/reference/block-kit/blocks#image Image block reference}.
  */
-export interface ImageBlock extends Block {
+type ImageBlock = {
```

Comment:
```suggestion
export type ImageBlock = {
```

Reply:
It is true that a union may only be assigned to a `type`, but the union `type` may be composed of interfaces.

For example, [this type for the `stars.*` API arguments](https://github.com/slackapi/node-slack-sdk/blob/664db4e58d5a7b7082d3006b969e82a4b459402e/packages/web-api/src/types/request/stars.ts#L17) contains a union that is composed of at least one interface (`StarsChannelDestination` is an interface).

Reply:
Yes of course. Ill get this fixed up later